### ISSUE TITLE
export/import database

### ISSUE THREAD

**Issue by JerryKrakel** [dwijitsolutions/laraadmin#125]
Hi There,

When i export a local dev database to my production server for some reason the image's won't work. Everything works fine just not the images. 

Now i'm adding the images again every time but thats taking a lot of time while the database gets bigger and bigger. Any solution for this?

**Comment by gdbhosale** [comment]
Hi,

You also have to upload the images stored in `storage/uploads` folder.
Please check details of [laraadmin.com/docs/1.0/uploads_manager](http://laraadmin.com/docs/1.0/uploads_manager).

Rgards,

**Comment by wasiqaftab** [comment]
@gdbhosale i did same but still i cant see images. i believe there is some links of image name which is loosed can you help me on this.

**Comment by JerryKrakel** [comment]
Still the same problem when i export to my production server. did upload the directory, did try it with git. images are there but are missing when loading the page..

Anyone else having this problem?

**Comment by wasiqaftab** [comment]
@JerryKrakel yes i have the same issue. try to solve it but didnt work. need to dig into code

**Comment by JerryKrakel** [comment]
Did you fix it already @wasiqaftab ? ;-)

**Comment by wasiqaftab** [comment]
@JerryKrakel still seeking help. Try to fix over weekend

**Comment by JerryKrakel** [comment]
U found a solution yet @wasiqaftab
